src/platform/packages/shared/presentation/presentation_publishing/publishing_subject/publishing_subject.test.tsx

```diff
@@ -244,7 +244,10 @@ describe('publishing subject', () => {
           </>
         );
       }
-      render(<Component />);
+      render(<Component />, {
+        // TODO: fails with concurrent mode
+        legacyRoot: true,
+      });
```

Comment:
This test fails without legacyRoot because it is expecting the `renderCount` to be 2 instead of 7. I suspect this is due to React batching updates. So maybe we should update the test assertion to be `2` and stop using `legacyRoot`? @ThomThomson @nreese

Reply:
Thanks! This is clearly one of those rare tests that checks React internals, like the number of renders. 
We can update this in the PR or follow up. Feel free to push directly if you'd like!

Reply:
I updated this test. 7 -> 2 makes sense